stetho/src/main/java/com/facebook/stetho/inspector/elements/AbstractChainedDescriptor.java

```diff
@@ -144,4 +144,14 @@ public final void setAttributesAsText(Object element, String text) {
   protected void onSetAttributesAsText(E element, String text) {
     mSuper.setAttributesAsText(element, text);
   }
+
+  @Override
+  @SuppressWarnings("unchecked")
+  public final void getStyleProperties(Object element, Accumulator<StyleInfo> accumulator) {
+    onGetStyles((E) element, accumulator);
+  }
+
+  protected void onGetStyles(E element, Accumulator<StyleInfo> accumulator) {
+    mSuper.getStyleProperties(element, accumulator);
```

Comment:
shouldn't this call to `mSuper.getStyleProperties(element, accumulator);` be done in `getStyleProperties` ? That way we get automatic aggregation of the super's style information. see also: `getChildren` and `onGetChildren` as well as `getAttributes` and `onGetAttributes`